easybuild/tools/py2vs3/py3.py

```diff
@@ -37,7 +37,6 @@
 import subprocess
 import sys
 import urllib.request as std_urllib  # noqa
-from collections import OrderedDict  # noqa
```

Comment:
We can't/shouldn't make a breaking change like this in EasyBuild 4.x, it will break any `from easybuild.tools.py2vs3 import OrderedDict` out there...

Reply:
Makes sense. Reverted that change